README.md

```diff
@@ -1,70 +1,23 @@
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `yarn start`
-
+# USWDS Practice
```

Comment:
Nice job writing a clear README! I like how you also explain your goals for the project.